src/test/groovy/graphql/analysis/QueryTraverserTest.groovy

```diff
@@ -1407,9 +1407,9 @@ class QueryTraverserTest extends Specification {
             """)
         QueryTraverser queryTraversal = createQueryTraversal(query, schema)
         QueryVisitorFieldEnvironment env
-        1 * visitor.visitField({ QueryVisitorFieldEnvironmentImpl it ->
-            env = it
-        })
+        1 * visitor.visitField(_) >> { args ->
```

Comment:
this is a change in spock (or maybe groovy) which doesn't allow assignment anymore in the argument matcher closure. Therefore we need to capture the arguments in the implementation closure